application/cards/card.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+class Card
+  attr_accessor :balance, :number, :type
+
+  CARD_TYPES = {
+    usual: 'usual',
+    capitalist: 'capitalist',
+    virtual: 'virtual'
+  }.freeze
+
+  NUMBER_OF_CARD_SIZE = 16
+  CARD_NUMBERS = 10
+
+  def initialize
+    @number = generate_number
+  end
+
+  def generate_number
+    Array.new(NUMBER_OF_CARD_SIZE) { rand(CARD_NUMBERS) }.join
+  end
+
+  def withdraw_tax(amount)
+    calculate_tax(amount, withdraw_percent_tax, withdraw_fixed_tax)
+  end
+
+  def put_tax(amount)
+    calculate_tax(amount, put_percent_tax, put_fixed_tax)
+  end
+
+  def sender_tax(amount)
+    calculate_tax(amount, sender_percent_tax, sender_fixed_tax)
+  end
+
+  def self.find_type(input)
+    CARD_TYPES.value?(input)
+  end
+
+  def put_money(amount)
+    @balance += amount - put_tax(amount)
+  end
+
+  def operation_put_valid?(amount)
+    amount >= put_tax(amount)
+  end
+
+  def withdraw_money(amount)
+    @balance -= amount - withdraw_tax(amount)
```

Comment:
you have attr reader `@balance` -> `balance`

Reply:
changed @balance -> balance

Reply:
removed "balance" because of rubocop and checked with rspec.